docs/pages/enroll-resources/auto-discovery/servers/gcp-discovery.mdx

```diff
@@ -93,6 +93,12 @@ discover instances.
   your custom role. Click *Save*.
 
   ![Role assignment](../../../../img/server-access/guides/gcp/assign-role-to-service-account@2x.png)
+
+  If the Discovery Services will run in a GCP compute instance, edit the instance and
+  assign the service account to the instance and set its Access Scopes to include 
```

Comment:
Is "Access Scopes" supposed to be a proper noun? (I.e., is it a GCP-specific feature?) If not, we can use "access scopes".